src/modules/edc-demo/components/contract-offer-list/contract-offer-list.component.html

```diff
@@ -12,7 +12,7 @@
           {{ asset.name }}
         </a>
       </mat-card-title>
-      <mat-card-subtitle>{{ asset.id }}</mat-card-subtitle>
+      <mat-card-subtitle>{{ asset.originatorOrganization }}</mat-card-subtitle>
```

Comment:
Missing "Unknown Organization". So you need it in two places, so you should find a way to not have to repeat yourself if possible.

If it's important to you that the "Unknown Organization" is in italics, you could create a component "asset-originator-organization-name".

Alternatively you could default the originatorOrganization to "Unknown Organization" in the asset mapper.